diff --git a/node_modules/binpack/.bun-tag-2d2c2d418238a255 b/.bun-tag-2d2c2d418238a255
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/index.js b/index.js
index d7ed315f898655019cc2e56be5e75352db5889ee..d59ca26b9005eb790415a3dd4f01a826508b624f 100644
--- a/index.js
+++ b/index.js
@@ -1,3 +1,6 @@
+// FIX
+var twoToThe32 = Math.pow(2, 32);
+
 // t is a binpack typename
 var sizeOfType = function(t) {
     // unsigned are the same length as signed
@@ -31,7 +34,7 @@ var addBindings = function(binpackTypename, nodeTypename){
         nodeTypename = binpackTypename;
     }
     module.exports['pack' + binpackTypename] = function(num, endian){
-        b = new Buffer(sizeOfType(binpackTypename));
+        var b = new Buffer(sizeOfType(binpackTypename));
         b['write' + nodeTypename + endianConv(endian, binpackTypename)](num, 0, true);
         return b;
     }
